include/pinocchio/multibody/joint/joint-free-flyer.hpp

```diff
@@ -190,6 +191,8 @@ namespace pinocchio
     typedef Eigen::Matrix<Scalar, NQ, 1, Options> ConfigVector_t;
     typedef Eigen::Matrix<Scalar, NV, 1, Options> TangentVector_t;
 
+    typedef boost::mpl::true_ is_mimicable_t;
```

Comment:
It should be false ?

Reply:
With the above comment, if no transform is specified, the joint can still be a mimic if $\alpha = 1, \beta = 0$. Thus we allow the freeflyer to be mimicked under this condition.